Dynatrace/dynatrace-operator
# Add `install-source` label to Operator deployment

## Description
Any value for `platform` is now allowed, with the default assumption that manifests for `kubernetes` are generated.
- Manifest and Olm are set as install-source if helm is run with those values
- Any other platform is assumed as special marketplace and passed to the label without modification
- Helm is the default value, if nothing else matches

> [K8S-10736](https://dt-rnd.atlassian.net/browse/K8S-10736)



## How can this be tested?

- Run `make test/helm`
- Try out different values for `platform` and check the resulting manifests are for the right platform and the label is set correctly
